frontend/src/app/shared/components/calendar/calendar.component.sass

```diff
@@ -20,5 +22,35 @@
 
 mat-label
     color: $gray-200
+    letter-spacing: 0
+    font-weight: 400
     font-size: 14px
 
+::ng-deep.mat-mdc-icon-button
+    --mat-mdc-button-persistent-ripple-color: $white
+    --mat-mdc-button-ripple-color: $blue-200
+
+.mat-mdc-form-field-input-control.mat-mdc-form-field-input-control
+    letter-spacing: 0
+    text-transform: none
+    font-size: 14px
+    font-weight: 400
+    font-style: normal
+    line-height: 146%
+
+::ng-deep.mat-mdc-form-field-flex
+    height: 36px !important
+
+::ng-deep.mat-mdc-form-field-infix
+    padding-top: 0 !important
+
+::ng-deep.mat-mdc-form-field-input-control
+    margin-top: 20px
+
+::ng-deep.mat-mdc-form-field-infix
+    min-height: 1px !important
+    height: 36px !important
+
+::ng-deep.mat-mdc-floating-label
+    top: 20px !important
```

Comment:
Have you done it according to design or just make it work?

Reply:
![image](https://user-images.githubusercontent.com/98846587/230040505-de0e2c13-dc6a-4f24-b60b-5fe9e868e0ac.png)
![image](https://user-images.githubusercontent.com/98846587/230040507-73ea2220-2ea2-4fe6-9e28-0406744bef21.png)
Without it label in placeholder will be down